core/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 
     <application>
         <activity
-            android:name="com.redmadrobot.debug_panel_core.ui.debugpanel.DebugActivity"
+            android:name="com.redmadrobot.debug.core.ui.debugpanel.DebugActivity"
```

Comment:
Here and in all others similar places, I'd suggest omitting namespace
```suggestion
            android:name=".ui.debugpanel.DebugActivity"
```

Reply:
https://github.com/RedMadRobot/debug-panel-android/pull/16#discussion_r1580774193